mdtraj/core/topology.py

```diff
@@ -1295,13 +1295,15 @@ class Chain(object):
         Iterator over all Atoms in the Chain.
     """
 
-    def __init__(self, index, topology):
+    def __init__(self, index, topology, chain_id = None):
```

Comment:
[pep8](https://www.python.org/dev/peps/pep-0008/) style:
```suggestion
    def __init__(self, index, topology, chain_id=None):
```

Reply:
funny, my pep8 was messing with the files and I had to turn it off. Done.